src/ui/blockUi-timeBased.js

```diff
@@ -28,51 +28,51 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
             onMediaEnded: null
         },
         invokers: {
-            updateMediaPlayer: {
-                "funcName": "sjrk.storyTelling.blockUi.timeBased.updateMediaPlayer",
-                "args": ["{that}.dom.mediaPlayer", "{arguments}.0"]
+            updateMediaPreview: {
+                "funcName": "sjrk.storyTelling.blockUi.timeBased.updateMediaPreview",
+                "args": ["{that}.dom.mediaPreview", "{arguments}.0"]
             },
             stopMediaPlayer: {
                 func: "{that}.events.onMediaPlayerStop.fire",
                 args: ["{that}"]
             },
             playMediaPlayer: {
                 "funcName": "sjrk.storyTelling.blockUi.timeBased.playMediaPlayer",
-                "args": ["{that}.dom.mediaPlayer"]
+                "args": ["{that}.dom.mediaPreview"]
             }
         },
         listeners: {
-            "onTemplateReady.updateMediaPlayerUrl": {
-                func: "{that}.updateMediaPlayer",
+            "onTemplateReady.updateMediaPreviewUrl": {
+                func: "{that}.updateMediaPreview",
                 args: ["{that}.block.model.mediaUrl"]
             },
             "onTemplateReady.bindOnMediaReady": {
-                "this": "{that}.dom.mediaPlayer.0",
+                "this": "{that}.dom.mediaPreview.0",
                 method: "addEventListener",
                 args: ["canplay", "{that}.events.onMediaReady.fire"]
             },
             "onTemplateReady.bindOnMediaDurationChange": {
-                "this": "{that}.dom.mediaPlayer.0",
+                "this": "{that}.dom.mediaPreview.0",
                 method: "addEventListener",
                 args: ["durationchange", "{that}.events.onMediaDurationChange.fire"]
             },
             "onTemplateReady.bindOnMediaEnded": {
-                "this": "{that}.dom.mediaPlayer.0",
+                "this": "{that}.dom.mediaPreview.0",
                 method: "addEventListener",
                 args: ["ended", "{that}.events.onMediaEnded.fire"]
             },
             "onTemplateReady.bindOnMediaPlay": {
-                "this": "{that}.dom.mediaPlayer.0",
+                "this": "{that}.dom.mediaPreview.0",
                 method: "addEventListener",
                 args: ["play", "{that}.events.onMediaPlay.fire"]
             },
```

Comment:
This should probably be collapsed into a single function to relay the necessary events.